File: app/src/main/java/com/tvisha/trooptime/activity/activity/Model/Notification.java
package com.tvisha.trooptime.activity.activity.Model;

public class Notification {
    public long getID() {
        return ID;
    }

    public void setID(long ID) {
        this.ID = ID;
    }

    public long ID;
    public String notificationdata;
    public int notificationtype;

    public String getnotificationdata() {
        return notificationdata;
    }

    public void setnotificationdata(String notificationdata) {
        this.notificationdata = notificationdata;
    }

    public int getnotificationtype() {
        return notificationtype;
    }

    public void setnotificationtype(int notificationtype) {
        this.notificationtype = notificationtype;
    }

    public String getCreated_at() {
        return created_at;
    }

    public void setCreated_at(String created_at) {
        this.created_at = created_at;
    }

    public String getUpdated_at() {
        return updated_at;
    }

    public void setUpdated_at(String updated_at) {
        this.updated_at = updated_at;
    }

    public String created_at;
    public String updated_at;
}

File: app/src/main/java/com/tvisha/trooptime/activity/activity/model/Notification.kt
package com.tvisha.trooptime.activity.activity.model

class Notification {
    var iD: Long = 0
    var notificationdata: String? = null
    var notificationtype = 0
    fun getnotificationdata(): String? {
        return notificationdata
    }

    fun setnotificationdata(notificationdata: String?) {
        this.notificationdata = notificationdata
    }

    fun getnotificationtype(): Int {
        return notificationtype
    }

    fun setnotificationtype(notificationtype: Int) {
        this.notificationtype = notificationtype
    }

    var created_at: String? = null
    var updated_at: String? = null
}